src/util.h

```diff
@@ -250,28 +250,23 @@ static SECP256K1_INLINE void secp256k1_int_cmov(int *r, const int *a, int flag)
     *r = (int)(r_masked | a_masked);
 }
 
-/* If USE_FORCE_WIDEMUL_{INT128,INT64} is set, use that wide multiplication implementation.
+/* If USE_FORCE_WIDEMUL_{INT128, INT128_STRUCT, INT64} is set, use that wide multiplication implementation.
  * Otherwise use the presence of __SIZEOF_INT128__ to decide.
```

Comment:
I think we should detect 64-bit MSVC here and select `SECP256K1_INT128_STRUCT`.

Reply:
Ok sure, that will make things easier. :+1:

Reply:
> I think we should detect 64-bit MSVC here and select `SECP256K1_INT128_STRUCT`.

Note for the future PR: What we should actually do is to use to word size to decide which implementation to use, independently of the exact compiler.